lightwood/encoders/time_series/plain.py

```diff
@@ -0,0 +1,34 @@
+import torch
+
+from lightwood.encoders.encoder_base import BaseEncoder
+from lightwood.constants.lightwood import COLUMN_DATA_TYPES
+from lightwood.encoders.time_series.helpers.common import MinMaxNormalizer, CatNormalizer
+
+
+class TimeSeriesPlainEncoder(BaseEncoder):
+    def __init__(self, is_target=False):
+        """
+        Fits a normalizer for a time series previous historical data.
+        When encoding, it returns a normalized window of previous data.
+        """
+        super().__init__(is_target)
+        self.original_type = None
+        self._normalizer = None
+
+    def prepare(self, priming_data):
+        if self._prepared:
+            raise Exception('You can only call "prepare" once for a given encoder.')
+        else:
+            if self.original_type == COLUMN_DATA_TYPES.CATEGORICAL:
+                self._normalizer = CatNormalizer()
+            else:
+                self._normalizer = MinMaxNormalizer()
+            self._normalizer.prepare(priming_data)
+        self._prepared = True
```

Comment:
just a coding style note (I thik it's in the guidelines)

when you have `if something -> return/raise` and the other branch is a long list of lines, don't but it in an `else` (I'm not inventing this, clang, pytorch and others this gudieline for readability and for python it's even more importnat due to indentation)

Reply:
Duly noted